INFO     backend.layers.business.business:business.py:553 {'message': 'ingesting dataset', 'collection_version_id': 375a64ed-9910-4909-b9cf-695981681692, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:553 {'message': 'ingesting dataset', 'collection_version_id': 375a64ed-9910-4909-b9cf-695981681692, 'url': None, 'current_dataset_version_id': 7b0a89d1-336b-4dd8-8544-985c36564da1}
INFO     processing:dataset_metadata_update.py:333 Main: No raw h5ad update required
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [db8c6d0a-06fd-4571-a683-f04de8b561a9/raw.h5ad] to s3://artifact_bucket/db8c6d0a-06fd-4571-a683-f04de8b561a9/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
ERROR    processing:dataset_metadata_update.py:353 Cannot find labeled H5AD artifact uri for 7b0a89d1-336b-4dd8-8544-985c36564da1.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:357 Main: Starting thread for rds update
INFO     processing:dataset_metadata_update.py:380 Main: Starting thread for cxg update
INFO     processing:process_logic.py:45 Updating processing status